Bottom: f3022b66106d7c358b4f1b3ff6fcee04c7d40020
Top:    b4b6c7283abf0adf94f01735fb51eef1de2f129d
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-30 15:46:16 +0100

autoroute.c: Temporarily turn on some debugging drawing


---

diff --git a/src/autoroute.c b/src/autoroute.c
index d752231..f34451f 100644
--- a/src/autoroute.c
+++ b/src/autoroute.c
@@ -92,16 +92,14 @@ RCSID ("$Id$");
 #define ROUTE_VERBOSE
 */
 
-/*
 #define ROUTE_DEBUG
-//#define DEBUG_SHOW_ROUTE_BOXES
+#define DEBUG_SHOW_ROUTE_BOXES
 #define DEBUG_SHOW_EXPANSION_BOXES
-//#define DEBUG_SHOW_EDGES
-//#define DEBUG_SHOW_VIA_BOXES
+#define DEBUG_SHOW_EDGES
+#define DEBUG_SHOW_VIA_BOXES
 #define DEBUG_SHOW_TARGETS
 #define DEBUG_SHOW_SOURCES
-//#define DEBUG_SHOW_ZIGZAG
-*/
+#define DEBUG_SHOW_ZIGZAG
 
 static direction_t
 directionIncrement(direction_t dir)
@@ -4181,7 +4179,7 @@ RouteOne (routedata_t * rd, routebox_t * from, routebox_t * to, int max_edges)
       /* we should never add edges on inactive layer groups to the heap. */
       assert (is_layer_group_active[e->rb->group]);
 #if defined(ROUTE_DEBUG) && defined(DEBUG_SHOW_EXPANSION_BOXES)
-      //showedge (e);
+      showedge (e);
 #endif
       if (e->rb->flags.is_thermal)
 	{
